lib/network.rb

```diff
@@ -0,0 +1,32 @@
+class Network
+  attr_reader :name, :shows, :characters
+  def initialize(name)
+  @name = name
+  @shows = []
+  @characters = characters
+  @payroll_sort = {}
+  end
+
+  def add_show(added_show)
+    @shows.push(added_show)
+  end
+
+  def payroll
+    payroll = {}
+    payroll["David Hasselhoff"] = 1600000
+    payroll["William Daniels"] = 1000000
+    payroll["Amy Poehler"] = 2000000
+    payroll["Nick Offerman"] = 1400000
+    payroll
```

Comment:
this will work excellently if these 4 actors are the only actors in any TV show ever :)